src/extension/src/CoreTSModule.ts

```diff
@@ -0,0 +1,33 @@
+import { WizardServant, IPayloadResponse } from "./wizardServant";
```

Comment:
NIT: The first character of the module name should not be capitalized, i.e. coreTSModule.ts

Reply:
PR updated :)